backend/.env.test

```
@@ -1,51 +1,83 @@
+K8S_POD_MEMORY_REQUEST=128Mi
+K8S_POD_EXECUTION_TIMEOUT=5
+RATE_LIMITS=100/minute
+RATE_LIMIT_ENABLED=false
 
-# Security
-SECURE_COOKIES=true
-BCRYPT_ROUNDS=4
+# Event-Driven Design Configuration
+KAFKA_BOOTSTRAP_SERVERS=kafka:29092
+SCHEMA_REGISTRY_URL=http://schema-registry:8081
+ENABLE_EVENT_STREAMING=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ENABLE_EVENT_STREAMING key should go before the KAFKA_BOOTSTRAP_SERVERS key

(UnorderedKey)